src/platform/datapath_winuser.c

```diff
@@ -1746,33 +1746,13 @@ QUIC_DISABLED_BY_FUZZER_END;
 
     if (QUIC_FAILED(Status)) {
         if (Socket != NULL) {
-            QuicTraceEvent(
-                DatapathDestroyed,
-                "[data][%p] Destroyed",
-                Socket);
             if (Socket->ProcsOutstanding != 0) {
-                for (uint16_t i = 0; i < SocketCount; i++) {
```

Comment:
This all scares me. When we get here, its possible there is outstanding completion status's on any number of the initialized queues.